### ISSUE TITLE
Error in the collar Tutorial especially in : cross_correlate(raw_store, config, cc_store)

### ISSUE THREAD

**Issue by Jurang-cloud** [mdenolle/NoisePy#128]
I was trying to do some cross correlation using the collar tutorial but unfortunately I couldn't get to work due to the errors I encountered. I have attached the collar notebook so that you can go through and see where probably I am making a mistake and needed to be corrected or if it is a genuine error from the notebook then you can fix it.

Thank you,
Job
[get_started.md](https://github.com/mdenolle/NoisePy/files/11570330/get_started.md)

**Comment by carlosgjs** [comment]
@Jurang-cloud - We've found there's an issue when running on Apple M1 computers, which is documented [here](https://docs.ray.io/en/master/ray-overview/installation.html#m1-mac-apple-silicon-support). The workaround is to install `grpcio` from conda:
```
pip uninstall grpcio; conda install grpcio=1.43.0 -c conda-forge
```

**Comment by Jurang-cloud** [comment]
Hi Dr. Carlos,
Thanks for your response, I will try that and let you know how it turns out. Again, I want to believe that I would be able to use the tutorial example with my own data to do correlation and stacking?


Cheers
Job
________________________________
From: carlosgjs ***@***.***>
Sent: Wednesday, May 31, 2023 2:42 PM
To: mdenolle/NoisePy ***@***.***>
Cc: Job Urang (Student) ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [mdenolle/NoisePy] Error in the collar Tutorial especially in : cross_correlate(raw_store, config, cc_store) (Issue #128)

CAUTION: This email originated from outside of the Colorado School of Mines organization. Do not click on links or open attachments unless you recognize the sender and know the content is safe.


@Jurang-cloud<https://github.com/Jurang-cloud> - We've found there's an issue when running on Apple M1 computers, which is documented here<https://docs.ray.io/en/master/ray-overview/installation.html#m1-mac-apple-silicon-support>. The workaround is to install grpcio from conda:

pip uninstall grpcio; conda install grpcio=1.43.0 -c conda-forge

**Comment by carlosgjs** [comment]
Yes, provided the data complies with the expected format and naming convention. I suggest inspecting the files created by the `download()` function. E.g.:

```
2019_02_01_12_00_00T2019_02_02_00_00_00.h5
2019_02_01_00_00_00T2019_02_01_12_00_00.h5
```
I.e. the .h5 files are expected to be named after the timespan of data they contain.

**Comment by Jurang-cloud** [comment]
Hi Dr. carlos,
The data files I have are basically .SAC files  (
EHE     2016-03-09 17:46:28.184 2016-03-26 16:20:28.185588867
).
Can I convert it to the .h files?


Cheers,
Job
________________________________
From: carlosgjs ***@***.***>
Sent: Wednesday, May 31, 2023 3:08 PM
To: mdenolle/NoisePy ***@***.***>
Cc: Job Urang (Student) ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [mdenolle/NoisePy] Error in the collar Tutorial especially in : cross_correlate(raw_store, config, cc_store) (Issue #128)

CAUTION: This email originated from outside of the Colorado School of Mines organization. Do not click on links or open attachments unless you recognize the sender and know the content is safe.


Yes, provided the data complies with the expected format and naming convention. I suggest inspecting the files created by the download() function. E.g.:

2019_02_01_12_00_00T2019_02_02_00_00_00.h5
2019_02_01_00_00_00T2019_02_01_12_00_00.h5


I.e. the .h5 files are expected to be named after the timespan of data they contain.
